### PR TITLE
Feat: Added Makefile

### PR DESCRIPTION
**Pull request by tanryberdi**
Added Makefile for testing purpose #2

### CODE REVIEW

**Comment by tanryberdi** [comment]
fix #2

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test:
+	@echo "Testing ..."
+	@go clean -testcache && go test ./...
```
Comment (vaguecoder) [inline Makefile:3]: Git warns about newline. Please add a newline at the end.

Comment (tanryberdi) [reply Makefile]: done

Comment (vaguecoder) [reply Makefile]: A couple of more changes here. Please add the following flags to go test

1. `-v` for verbosity
2. `-race` for data race detection
3. `-coverprofile cover.out` for coverage profile

With `cover.out`, we can get the coverage. Please add this in next line.
```
go tool cover -func cover.out
```
Also, `go test` needn't be hid from stdout. You may keep `go clean` in `@` and take `go test` command to next line.

Comment (tanryberdi) [reply Makefile]: done

**Comment by vaguecoder** [comment]
LGTM. Thanks @tanryberdi

**Pull request merged**